x-pack/qa/sql/src/main/java/org/elasticsearch/xpack/qa/sql/jdbc/SqlSpecTestCase.java

```diff
@@ -60,8 +60,9 @@ public SqlSpecTestCase(String fileName, String groupName, String testName, Integ
 
     @Override
     protected final void doTest() throws Throwable {
-        boolean goodLocale = !(Locale.getDefault().equals(new Locale.Builder().setLanguageTag("tr").build())
-                || Locale.getDefault().equals(new Locale.Builder().setLanguageTag("tr-TR").build()));
+        String[] badLocales = new String[] {"tr", "az", "tr-TR", "tr-CY", "az-Latn", "az-Cyrl", "az-Latn-AZ", "az-Cyrl-AZ"};
```

Comment:
These locales are not bad :)  it's just some application are buggy and cannot work with them correctly. So, maybe we should call them `h2IncompatibleLocales` or `dottedILocales` or something like this.